assets/prefabs/Cucumber/Cucumber.prefab

```diff
@@ -14,5 +14,11 @@
     "bud": "AdditionalVegetables:CucumberBud",
     "minGrowTime": 4000,
     "maxGrowTime": 7000
+  },
+  "Food": {
```

Comment:
I just noticed that these components are available as transitive dependencies via EdibleFlora. I'm wondering whether we should make Hunger and Thirst explicit (optional) dependencies to reflect this better :thinking:

... and I'm pretty sure this is not the only place where we are using components of transitive dependencies.